scripts/generate-app-config/generateSearchItems.js

```diff
@@ -0,0 +1,21 @@
+/**
+* Generates the file search items.
+*/
+const generateSearchItems = (contentConfig) => {
+  const content = contentConfig.config;
+  const config = Object.keys(content).reduce((acc, navItemKey) => {
```

Comment:
Would it be simpler to use Object.entries ? I saw @ryanthemanuel use it recently and it's pretty slick

I think we'd end up with something like:

```
const config = []
Object.entries(content).forEach(([navItemKey, navItem]) = > {
     Object.entries(navItem).forEach(([pageKey, page]) => {
          config.push({
             title: page.name,
             tags: page.path.replace(/\//gi, ' '),
             path: page.path,
           });
     });
});
```

Reply:
`Object.entries` is polyfilled by core-js, so there shouldn't be any compat issues.